.github/test-config.yaml

```diff
@@ -0,0 +1,18 @@
+namespace: integration-temp
+chart-dir:
+  - charts
+excluded-charts:
+  - charts/sqlbackup
+  - charts/install-secrets-manager
+  - charts/cron-job
+  - charts/datarepomonitoring
+  - charts/gcp-managed-cert
+charts:
+  - charts/backend-config
+  - charts/create-secret-manager-secret
+  - charts/datarepo
+#  - charts/datarepo-api
```

Comment:
Did you mean to leave these?

Reply:
I will move them to excluded as the individual charts cannot be tested by itself but the umbrella can be selected to be tested.